docs/content/guides/columns/column-width.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Column width
-metaTitle: Column width - Guide - Handsontable Documentation
+metaTitle: Column width - JavaScript Data Grid | Handsontable
+description: Configure column widths, using an array or a function. Let your users manually change column widths using Handsontable's interface.
```

Comment:
```suggestion
description: Configure column widths using an array or a function. Let your users manually change column widths using Handsontable's interface.
```

Reply:
@bulaj – you're right that commas like this are rare in everyday English (that's why tools like Grammarly usually point it out), but they [reduce ambiguity](https://ell.stackexchange.com/questions/47134/a-comma-before-using) (e.g., here, without the comma, we've got 2 possible meanings, with the comma, just one)